### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by TimaliveProgman**

### CODE REVIEW

**Comment by TimaliveProgman** [comment]
send data with riverpod

File: output.txt
Comment (denis-progman) [inline output.txt:1]: What is this? do we really need this file?

File: lib/providers/auth_providers.dart [hunk 1]
```diff
@@ -23,4 +23,4 @@ final resetStorage = StateProvider<dynamic>(
     final isCleared = await prefs.clear();
```
Comment (denis-progman) [inline lib/providers/auth_providers.dart:23]: prefs - is an unclear variable name. let's don't cut down words. the same thing about the name of the provider. 
Please rename it completely understandable and logicaly.

**Review by denis-progman — COMMENTED** [review]
somehow I don't see all the changes connected with the provider usage. 
So I cannot check all of your work. we need to discuss it at least in the TG chat.

**Review by denis-progman — CHANGES_REQUESTED** [review]
Please fix all of the comments and comment back.

File: output.txt
Comment (TimaliveProgman) [reply output.txt:1]: i don't khow, I see it for the first time

Comment (TimaliveProgman) [reply output.txt:1]: file deleted

File: lib/providers/auth_providers.dart (hunk 1 above)
Comment (TimaliveProgman) [reply lib/providers/auth_providers.dart:23]: changed

**Review by denis-progman — APPROVED** [review]

**Pull request merged**